src/Test/L1/Tasks/d9bafed4-0b18-4f58-968d-86655b4d2ce9/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -0,0 +1,18 @@
+{
+  "loc.friendlyName": "Command line",
```

Comment:
what's this? why are there localization changes in this PR?

Reply:
Its part of the task, but we don't need it - removed.